### PR TITLE
Move pulpcore-plugin into pulpcore as a subpackage

### PR DESCRIPTION
**Pull request by dralley**
re: #5580
https://pulp.plan.io/issues/5580

### CODE REVIEW

File: templates/travis/.travis.yml.j2 [hunk 1]
```diff
@@ -11,23 +11,23 @@ python:
   - "3.7"
 env:
   matrix:
-    - DB=postgres TEST=pulp
```
Comment (dralley) [inline templates/travis/.travis.yml.j2]: Leftover from MariaDB support, don't need it anymore.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,3 @@
-<!-- TEMPLATE_REMOVE_START -->
```
Comment (dralley) [inline README.md]: Leftover from when we used bootstrap.py

File: templates/travis/.travis/before_install.sh.j2 [hunk 3]
```diff
@@ -91,27 +89,17 @@ fi
...
-
-  git clone --depth=1 https://github.com/PulpQE/pulp-smash.git
-
-  if [ -n "$PULP_SMASH_PR_NUMBER" ]; then
-    cd pulp-smash
-    git fetch --depth=1 origin +refs/pull/$PULP_SMASH_PR_NUMBER/merge
-    git checkout FETCH_HEAD
-    cd ..
-  fi
+if [ -n "$PULP_SMASH_PR_NUMBER" ]; then
```
Comment (mikedep333) [inline templates/travis/.travis/before_install.sh.j2]: Why was the indentation collapsed?

Comment (dralley) [reply templates/travis/.travis/before_install.sh.j2]: No particular reason other than it bothered me when I saw the output in pulp_file.  I can reverse it

**Pull request merged**